local-cli/generator-ios/templates/xcodeproj/project.pbxproj

```diff
@@ -479,6 +510,20 @@
 			remoteRef = 146834031AC3E56700842450 /* PBXContainerItemProxy */;
 			sourceTree = BUILT_PRODUCTS_DIR;
 		};
+        5E9157331DD0AC6500FF2AA8 /* libRCTAnimation.a */ = {
+            isa = PBXReferenceProxy;
+            fileType = archive.ar;
+            path = libRCTAnimation.a;
+            remoteRef = 5E9157321DD0AC6500FF2AA8 /* PBXContainerItemProxy */;
+            sourceTree = BUILT_PRODUCTS_DIR;
+        };
+        5E9157351DD0AC6500FF2AA8 /* libRCTAnimation-tvOS.a */ = {
```

Comment:
All libraries should have two targets now, one for iOS and one for tvOS.  So this is probably correct.  If you have test automation for the new library, and it is passing on tvOS, then the project is probably correct :)